src/components/ConsoleOptions/index.js

```diff
@@ -62,6 +62,9 @@ class ConsoleOptions extends React.Component {
             />
           </dd>
 
+          <div className={style['console-client-resources']}>
+            Refer <a href='https://www.ovirt.org/documentation/admin-guide/virt/console-client-resources/' target='_blank'>Console Client Resources</a> in case of troubles.
```

Comment:
Tested on version: commit 252b932, components commit c00b232

* Console block is nicely aligned in contrast to disks: I find disks unnecessarily shifted to the right and a bit illogically starting at the same line as the label.
* When "hide" link is clicked in Console section then the hide-able block jumps left.
* I'd suggest to use arrows down and right in accordance with [1].

[1]: https://www.patternfly.org/pattern-library/forms-and-controls/expand-collapse-section/#/overview